src/validation.h

```diff
@@ -356,7 +356,31 @@ struct CBlockIndexWorkComparator
  * This data is used mostly in `CChainState` - information about, e.g.,
  * candidate tips is not maintained here.
  */
-class BlockManager {
+class BlockManager
+{
+    friend CChainState;
+
+private:
+    /* Calculate the block/rev files to delete based on height specified by user with RPC command pruneblockchain */
```

Comment:
(Only if you retouch the branch again) Make this a doxygen comment (start with `/**`). Also line-wrap this comment and the one below at something sensible :slightly_smiling_face: